log/core_wrapper.go

```diff
@@ -17,7 +30,10 @@ func (w wrappedCore) With(fields []zapcore.Field) zapcore.Core {
 }
 
 func (w wrappedCore) Write(entry zapcore.Entry, fields []zapcore.Field) error {
-	return w.Core.Write(entry, wrapFields(fields))
+	start := time.Now()
+	err := w.Core.Write(entry, wrapFields(fields))
+	logLatencySeconds.Observe(time.Since(start).Seconds())
+	return err
```

Comment:
🔕/nit: you can use `defer` so you don't even need to change the original line:
```suggestion
	defer func(start time.Time) {
		logLatencySeconds.Observe(time.Since(start).Seconds())
	}(time.Now())
	return w.Core.Write(entry, wrapFields(fields))
```